core/database/delta.go

```diff
@@ -25,3 +25,12 @@ func (d Delta) Stmt() string {
 func (d Delta) Args() []any {
 	return d.args
 }
+
+// Merge will merge the supplied Delta on to the end of this Delta returning a
+// new combined delta.
+func (d Delta) Merge(delta Delta) Delta {
```

Comment:
This is going to get you into trouble in many cases. Bind args don't play nicely with multi-statement blocks.
There's no need for it anyway; you can just have another delta in the list, right?

Reply:
Thanks. Make sense. In my head I was trying to keep them together for logic reasons. I've added them to the schemas list now. How do we feel about that?